databus-connector/src/main/java/com/sequenceiq/cloudbreak/sigmadbus/config/SigmaDatabusConfig.java

```diff
@@ -4,8 +4,10 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
+import com.sequenceiq.cloudbreak.streaming.config.AbstractStreamingConfiguration;
+
 @Component
-public class SigmaDatabusConfig {
+public class SigmaDatabusConfig extends AbstractStreamingConfiguration {
```

Comment:
as this config is a bit more "global" (as shared between multiple processors), probably it's not a good idea to use this as an implementation of abstractStreamConfiguration, maybe still we should have an AbstractDatabusConfiguration that extends AbstractStreamingConfiguration, and uses this SigmaDatabusConfig as well (by incjection, so using only one instance for this)